tests/cli/output/help.txt

```diff
@@ -69,6 +69,6 @@ Note: Use comments containing ' *INDENT-OFF*' and ' *INDENT-ON*' to disable
       processing of parts of the source file (these can be overridden with
       enable_processing_cmt and disable_processing_cmt).
 
-There are currently 650 options and minimal documentation.
+There are currently 651 options and minimal documentation.
```

Comment:
master might be broken if you have to change this here, don't you think?

Reply:
This was fixed by #1940, so I guess this PR could be rebased now...

Reply:
Is done. Please review again.

Reply:
It looks like you merged, rather than rebasing. Unfortunately, that is making github think this PR now has 22 commits...

Reply:
What is to do?

Reply:
```bash
$ git checkout pp_multi_comment
$ git rebase master
$ git push guy-maurel --force-with-lease HEAD
```
(Replace `guy-maurel` with whatever is the name of the remote of your fork.)

If you `git log --decorate --graph` before pushing, you should see one (or maybe a couple) *non-merge* commits after `master`.